helm/archive-node/templates/archive.yaml

```diff
@@ -110,6 +110,7 @@ spec:
         args: [ 
           "coda-archive",
           "run",
+          "-metrics-port", "{{ .Values.archive.ports.metrics }}",
```

Comment:
Yikes looks like only a single port definition is supported per pod/deployment https://github.com/prometheus/prometheus/issues/3756 (see port annotations at top of file).

Thinking we may be able to just use the archive metrics port (instead of the daemon) or add a condition based on input vars.